src/axom/quest/examples/containment_driver.cpp

```diff
@@ -386,15 +508,25 @@ struct Input
 #endif
   }
 
+  bool isInput2D() const
+  {
+    using axom::utilities::string::endsWith;
+    if(endsWith(inputFile, ".contour"))
```

Comment:
This is just a small style thing, but I prefer

```
return predicate();
```

to

```
if (predicate()) {
    return true;
}
return false;
```

Reply:
Thanks -- me too.

I wasn't sure if I'd need more development there, so I left it more general at firstl.
Since I didn't actually need the flexibility, I've adopted your suggestion